VAMobile/src/utils/waygateConfig.test.ts

```diff
@@ -0,0 +1,12 @@
+const mockSetDefaultsSpy = jest.fn()
+
+jest.mock('@react-native-firebase/remote-config', () => () => ({
+    fetch: jest.fn(() => Promise.resolve()),
+    getValue: jest.fn(() => ({
+      asBoolean: () => true,
+    })),
+    getAll: jest.fn(() => false),
+    activate: jest.fn(() => Promise.resolve()),
+    setConfigSettings: jest.fn(() => Promise.resolve()),
+    setDefaults: mockSetDefaultsSpy,
+  }))
```

Comment:
I don't think this file is needed. Or if you are keeping it you need to add some unit tests